frontend/src/components/admin/view-assessments/AssessmentsTable.tsx

```diff
@@ -36,6 +40,12 @@ const AssessmentsTable = ({
         assessmentStatus={assessment.status}
       />
     ),
+    onClick: () => {
+      if (checkFeatureFlag("ENABLE_ADMIN_DATA_VIZ")) {
+        // TODO fill this in
+        history.push("TBD");
```

Comment:
can we create a ticket to track this work?

Reply:
@joyce-shi isn't that just the entire admin data viz stuff? Or you wanted an additional ticket for this specifically? I imagined whoever starts on admin data viz would do it since it kinda blocks all of that

Reply:
I guess that's fair